apps/site/pages/en/about/get-involved/index.md

```diff
@@ -5,9 +5,11 @@ layout: about
 
 # Get Involved
 
+If you are interested in getting involved with the Node.js community, there are many ways to do so. The Node.js project is a large and diverse community, and there are many ways to contribute, beyond just writing code.
+
 ## Community Discussion
 
-- The [GitHub issues list](https://github.com/nodejs/node/issues) is the place for discussion of Node.js core features.
+- The [GitHub issues list](https://github.com/nodejs/node/issues) is the place for discussion of Node.js core features and if you have questions about node.js, you can use the [github discussions](https://github.com/orgs/nodejs/discussions).
```

Comment:
Could you include nodejs/help for community support? I know this is about getting involved, but letting people know there is a dedicated help repo would IMO be useful

Reply:
True that isn't referenced on node.js website